doc/modules/preprocessing.rst

```diff
@@ -698,16 +677,15 @@ It is possible to adjust the threshold of the binarizer::
          [1., 0., 0.],
          [0., 0., 0.]])
 
-As for the :class:`StandardScaler` and :class:`Normalizer` classes, the
-preprocessing module provides a companion function :func:`binarize`
-to be used when the transformer API is not necessary.
+
+See also :func:`binarize`.
```

Comment:
Same here, but for `sklearn.preprocessing.StandardScaler` and `sklearn.preprocessing.Normalizer`.

Reply:
o